test/extended/cluster/cl.go

```diff
@@ -254,7 +258,7 @@ func mkPath(filename, config string) (string, error) {
 	}
 	// Handle an empty filename.
 	if filename == "" {
-		return "", fmt.Errorf("No template file defined!\n")
+		return "", fmt.Errorf("no template file defined!")
```

Comment:
Agree with removing '\n', not so sure about '^A-Z' -> '^a-z', but don't care that much. However, if we're changing this at some place, we should be consistent and the same rules should apply across all of cluster-loader.  Perhaps a follow-up cleanup PR if you don't want to do this in this one?

Reply:
Ah the reason was that golint said to update it, I will change the rest as well. Thanks.